testing/trino-product-tests/conf/product-tests-config-hive3.sh

```diff
@@ -0,0 +1,3 @@
+export HADOOP_BASE_IMAGE="ghcr.io/trinodb/testing/hive3.1-hive"
```

Comment:
Remove this file. This is a very old (pre-PTL) way of doing environment Configs

Reply:
just to make sure, do you mean `testing/trino-product-tests/conf/product-tests-config-hive3.sh` not `ghcr.io/trinodb/testing/hive3.1-hive` ?

Reply:
sorry for not being clear. remove the `testing/trino-product-tests/conf/product-tests-config-hive3.sh` file.

the `testing/trino-product-tests/conf/product-tests-config-hive3.sh` file is equivalent of the Config class, from times when PTL was just a bunch of bash scripts.

Reply:
> Remove this file. This is a very old (pre-PTL) way of doing environment Configs


I was wrong.  This can be used eg here https://github.com/trinodb/trino/blob/7d7df75fb7561a92dc0bd231f53a66113419af6b/.github/workflows/ci.yml#L138
Let's add this new config to `hive-tests` in ci.yml (if it works; otherwise commented out, with a TODO comment).

See also https://github.com/trinodb/trino/pull/9136

Reply:
thanks for pointing that out !